pcweb/pages/docs/api_reference/event_triggers.py

```diff
@@ -205,6 +227,7 @@ def change_text(self):
     },
 ]
 
+exec("from datetime import datetime")
```

Comment:
Can't this just be a normal import (without an exec?)

Reply:
#195